README.md

```diff
@@ -30,18 +30,53 @@
 </a>
 <br/>
 <a href="https://play.google.com/store/apps/details?id=org.jellyfin.androidtv">
-<img width="200" src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png" alt="Get it on Google Play"/>
+<img width="153" alt="Jellyfin on Google Play" src="https://jellyfin.org/images/store-icons/google-play.png"/>
 </a>
-<br/>
 <a href="https://www.amazon.com/gp/aw/d/B07TX7Z725">
-<img width="170" src="https://images-na.ssl-images-amazon.com/images/G/01/mobile-apps/devportal2/res/images/amazon-appstore-badge-english-black.png" alt="Available at Amazon Appstore"/>
+<img width="153" alt="Jellyfin on Amazon Appstore" src="https://jellyfin.org/images/store-icons/amazon.png"/>
 </a>
 <br/>
 <a href="https://repo.jellyfin.org/releases/client/androidtv/">Download archive</a>
 </p>
 
-Jellyfin Android TV is a Jellyfin client for Android TV, Nvidia Shield, and Amazon Fire TV devices. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start. Translations can be improved very easily from our <a href="https://translate.jellyfin.org/projects/jellyfin/jellyfin-androidtv">Weblate</a> instance. Look through the following graphic to see if your native language could use some work!
+Jellyfin Android TV is a Jellyfin client for Android TV, Nvidia Shield, and Amazon Fire TV devices.
+We welcome all contributions and pull requests! If you have a larger feature in mind please open an
+issue so we can discuss the implementation before you start. 
+
+## Translating
+Translations can be improved very easily from our
+[Weblate](https://translate.jellyfin.org/projects/jellyfin/jellyfin-androidtv) instance.
+Look through the following graphic to see if your native language could use some work!
 
-<a href="https://translate.jellyfin.org/engage/jellyfin/?utm_source=widget">
-<img src="https://translate.jellyfin.org/widgets/jellyfin/-/jellyfin-androidtv/multi-auto.svg" alt="Detailed Translation Status"/>
+<a href="https://translate.jellyfin.org/engage/jellyfin/">
+<img alt="Detailed Translation Status" src="https://translate.jellyfin.org/widgets/jellyfin/-/jellyfin-androidtv/multi-auto.svg"/>
 </a>
+
+## Build Process
+
+### Dependencies
+
+- Android SDK
+
+### Build
+
+1. Clone or download this repository
+
+   ```sh
+   git clone https://github.com/jellyfin/jellyfin-androidtv.git
+   cd jellyfin-androidtv
+   ```
+
+2. Open the project in Android Studio and run it from there or build an APK directly through Gradle:
+
+   ```sh
+   ./gradlew assembleDebug
```

Comment:
shouldn't this be 

ANDROID_SDK_ROOT={your_sdkPath} ./gradlew assembleDebug
```suggestion
   ANDROID_SDK_ROOT={AndroidSDK_path} ./gradlew assembleDebug
```

Reply:
I added a note that you need to add the Android SDK to your PATH env.